src/main/java/com/google/api/codegen/GapicContext.java

```diff
@@ -90,4 +94,17 @@ public CollectionConfig getCollectionConfig(Interface service, String entityName
     }
     return result;
   }
+
+  /**
+   * Returns a list of simple RPC methods.
+   */
+  public List<Method> getSimpleRpcMethods(Interface service) {
```

Comment:
Ahh! Very good point and I agree. I think there might be a better name than "simple" though. In the MVVM view model layer, for Java methods, I am using "simple" to contrast with page streaming and bundling.